mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -233,6 +243,8 @@ trait OverrideCompletions { this: MetalsGlobal =>
       val name: String = Identifier(sym.name)
 
       val filterText: String = s"${overrideKeyword}${lzy}${keyword}${name}"
+      val insertText: String =
+        s"${overrideKeyword}${modifs}${keyword}${name}$signature"
```

Comment:
Will this work if we want to override lazy vals? They cannot be abstract but we can override them and this compilation is also used for non-abstract symbols.

Reply:
lazy should be in modifs already, I wanted to just get rid of lzy and only use modifs but it is used to match against what user typed and matching against modifiers is probably not what user wants.
Gonna double check lazy is there :+1:

Reply:
Cool, let's add a test if there is none.

Reply:
yes, there is already a test :+1:
```
  checkEditLine(
    "lazy",
    """|package q
       |abstract class Val {
       |  lazy val hello: Int = 2
       |}
       |class Main extends Val {
       |   ___
       |}
       |""".stripMargin,
    "override val hel@@",
    "override lazy val hello: Int = ${0:???}"
  )
```